pkg/util/util.go

```diff
@@ -114,11 +114,11 @@ func GetParent(client *kubernetes.Client, meta metav1.ObjectMeta) (string, bool)
 				if vw.OwnerReferences != nil {
 					return GetParent(client, vw.ObjectMeta)
 				}
-				return "ValidatingWebhook/" + vw.Name, false
+				return "ValidatingWebhook/" + vw.Name, true
 			}
 		}
 	}
-	return meta.Name, false
```

Comment:
I'm not sure about this line . can you help

Reply:
@AlexsJones  , can you help ?

Reply:
sorry @panpan0000 just saw it,
yes, I think an empty string makes more sense, the meta.Name doesn't give any more information in the output
 e.g
`return "", false`

Reply:
This should return false

Reply:
thank you both, It's fixed.